src/item.cpp

```diff
@@ -2608,35 +2620,112 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
-                                          encumbrance_when_full ) );
+            if( !t->data.empty() ) {
+                struct translation_pair {
+                    translation to_display;
+                    translation multiple_name;
+                };
+                struct armor_portion_type {
+                    int encumber;
+                    int max_encumber;
+                    int coverage;
+
+                    bool operator==( const armor_portion_type &other ) {
+                        return encumber == other.encumber
+                               && max_encumber == other.max_encumber
+                               && coverage == other.coverage;
+                    };
+                };
+                struct bodypart_info_pair {
+                    armor_portion_type portion;
+                    translation_pair translations;
+                };
+
+                std::map<bodypart_str_id, bodypart_info_pair> piece_data;
+
+                for( const armor_portion_data &piece : t->data ) {
+                    if( piece.covers.has_value() ) {
+                        for( const bodypart_str_id &covering_id : piece.covers.value() ) {
+                            if( covering_id != bodypart_str_id( "num_bp" ) ) {
+                                piece_data.insert( {
+                                    covering_id, { {
+                                            get_encumber( player_character, covering_id ),
+                                            get_encumber( player_character, covering_id, encumber_flags::assume_full ),
+                                            piece.coverage
+                                        }, {
+                                            covering_id.obj().name_as_heading,
+                                            covering_id.obj().name_as_heading_multiple
+                                        }
+                                    }
+                                } );
+                            }
+                        }
+                    }
+                }
+                // Handle things that use both sides to avoid showing L. Arm R. Arm etc when both are the same
+                if( !t->sided ) {
+                    // Cannot remove from a container we are iterating through so store what to remove then remove it afterwards
+                    std::vector<bodypart_str_id> to_remove;
+                    for( const body_part &legacy_part : all_body_parts ) {
+                        bodypart_str_id bp( convert_bp( legacy_part ) );
+                        bodypart_str_id opposite = bp->opposite_part;
+                        if( opposite != bp && covers( bp ) && covers( opposite )
+                            && piece_data.at( bp ).portion == piece_data.at( opposite ).portion ) {
+                            const bodypart_str_id mult_name = bodypart_str_id( bp->name_as_heading_multiple.translated() );
```

Comment:
This line is very suspicious.  I can't imagine any good reason to use a translated name as a `bodypart_str_id`.

Reply:
I am not sure how else to get a pluralised string in order to show the player in this scenario. In hindsight it won't work in any language other than english.

Reply:
My instinct is that `translation_pair` should be replaced by a simple `translation` object, which is initially assigned the singular name.  When you merge two opposite bodyparts that object (in the not-erased half) can be replaced (assigned to) with the multiple name.